.storybook/webpack.config.js

```diff
@@ -95,5 +106,7 @@ module.exports = ({ config }) => {
     },
   ];
 
+  config.optimization.minimizer[0].options.sourceMap = disableSourceMaps;
```

Comment:
hm,  but we just set minimizer to empty array. How can we access this property now?

Reply:
No. We keep the minimizer in my commit. We just disable source maps emission.

Reply:
I intended to actually use terser but it to not emit any source maps. Fixed up my commit.